bitrise.yml

```diff
@@ -150,7 +150,7 @@ workflows:
     - deploy-to-bitrise-io@1: {}
     meta:
       bitrise.io:
-        stack: osx-xcode-13.0.x
+        stack: osx-xcode-13.1.x
```

Comment:
If build is green I will try with xcode 13.2.x and if all green modify all the different stacks not only the testing workflows for that version latest version or this previous one